### PR TITLE
Dev ringwraiths

### PR DESCRIPTION
**Pull request by lendavhollandlane**
removed texts from ringwraiths and nazgul killed events.
Replaced randomturns with turn intervals in some events
Created event that causes war between Sauron and swarthy men on certain conditions. This is tested.
mordor and sm war event test

### CODE REVIEW

**Comment by markourm** [comment]
how this was tested?

**Comment by lendavhollandlane** [comment]
On turn 1 I created 3 dragons owned by Swarthy men near each of the target cities and attacked the cities and captured them. This should have put all 3 flags up.
On turn 2 I received a message that alliance between mordor and sw men is cancelled and then event text appeared that confirmed that Sauron has betrayed us.

**Comment by markourm** [comment]
untrustworthy maggot as he is

File: SWEvents.txt [hunk 1]
```diff
@@ -920,6 +922,21 @@ who=Swarthy men
...
 
+@IF
+Checkflag
+Who=Swarthy men
+Mask=0b00000000000000000000000111000000
+Count=3
+State=On
+@THEN
+MAKEAGGRESSION
+who=Mordor
```
Comment (markourm) [inline SWEvents.txt]: aggressor should be Swarthy men

Comment (lendavhollandlane) [reply SWEvents.txt]: thank you, will fix asap

**Comment by lendavhollandlane** [comment]
Fixed now.

**Review by markourm — APPROVED** [review]

**Pull request merged**